src/test/java/nextstep/fp/CarTest.java

```diff
@@ -8,24 +8,14 @@
     @Test
     public void 이동() {
         Car car = new Car("pobi", 0);
-        Car actual = car.move(new MoveStrategy() {
-            @Override
-            public boolean isMovable() {
-                return true;
-            }
-        });
+        Car actual = car.move((MoveStrategy)() -> true);
```

Comment:
```suggestion
        Car actual = car.move(() -> true);
```
굳이 타입 변환을 하지 않아도 됩니다!

Reply:
으아..저는 이것저것 찾아보다가 명시해주는게 좋을것 같다고 생각이 되어서 넣어본건데.. 생략하는게 더 코드상 좋은건지요!